package.json

```diff
@@ -36,6 +38,8 @@
     "@nx-extend/shadcn-ui": "^2.0.3",
     "@nx-tools/nx-container": "^6.0.1",
     "@nx/devkit": "19.3.1",
+    "@nx/eslint": "19.3.0",
```

Comment:
**blocker:** For some reason Nx keeps adding ESLint, we should get rid of it. Please remove all packages related to ESLint - we use Biome.